tests/storage/datavolume.go

```diff
@@ -350,48 +350,6 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 				libstorage.DeleteDataVolume(&dataVolume)
 			})
 
-			It("should accurately report DataVolume provisioning", func() {
```

Comment:
Are we sure this is redundant? At the very least functests exercise the actual API server interaction (watches, conflicts etc)

Reply:
Never thought it will go without a fight lol
There are so many tests that do this exact scenario, the only different thing that is being checked here is the status which is not really dependent in any API stuff, the UT covers the expected printable status just fine